server/src/AppBundle/Notifications/NotificationFactory/ProductEmailDeployment/OptIn.php

```diff
@@ -27,7 +27,7 @@ public function generate(Model $submission, Model $template = null, array $args)
         }
         foreach ($submission->getStore()->findQuery('product', ['_id' => ['$in' => $ids]]) as $product) {
             $id = (string) $product->getId();
-            $modifiedSubscriptions[$id] = ['name' => $product->get('name'), 'status' => ($optIns[$id] && $optIns[$id] !== false) ? 'Subscribed' : 'Unsubscribed'];
+            $modifiedSubscriptions[$id] = ['name' => $product->get('name'), 'status' => ($optIns[$id] && ($optIns[$id] !== false || $optIns[$id] !== "false")) ? 'Subscribed' : 'Unsubscribed'];
```

Comment:
I'd drop the or and just leave the string false check

Reply:
Done.